test/parallel/test-vm-preserves-property.js

```diff
@@ -0,0 +1,24 @@
+var common = require('../common');
```

Comment:
Fixing lint errors now, feeling dumb. Surprisingly none of the other tests have use strict but the linter still wants it.